concent_api/core/migrations/0004_auto_20180620_0712.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
Migrations should be squashed if possible.

Reply:
Done.

Reply:
There are still two migration files.

Reply:
There are two migrations in separate commits, because each commit have changes to models

Reply:
I that case it's fine.